injected/src/features/duckplayer/overlays.js

```diff
@@ -27,12 +27,12 @@ export async function initOverlays(settings, environment, messages) {
     try {
         initialSetup = await messages.initialSetup();
     } catch (e) {
-        console.error(e);
+        console.warn(e);
```

Comment:
I've lowered the severity of these log messages because during the experiment, Duck Player will need to exit gracefully if it's not needed. The way it knows that is by not receiving a response to initialSetup. Keeping this as an error was throwing off other people looking at the logs.